frontend/src/components/verticalPodAutoscaler/Details.tsx

```diff
@@ -0,0 +1,174 @@
+              <Link kubeObject={item.referenceObject}>
+                {item.referenceObject?.kind}/{item.referenceObject?.metadata.name}
+              </Link>
+            ),
+          },
+          {
+            name: t('glossary|Update Policy'),
+            value: (
+              <>
+                {t('glossary|Update Mode')}:{item.spec.updatePolicy.updateMode}
+              </>
+            ),
+          },
+        ]
+      }
+      extraSections={item =>
+        item && [
+          {
+            id: 'headlamp.vpa-container-policy',
+            section: (
+              <>
+                <SectionHeader title={t('glossary|Container Policy')} />
+                <SimpleTable
+                  data={item.spec.resourcePolicy?.containerPolicies}
+                  columns={[
+                    {
+                      label: t('glossary|Container'),
+                      getter: item => item.containerName,
+                    },
+                    {
+                      label: t('translation|Controlled Resources'),
+                      getter: item => item.controlledResources.join(', '),
+                    },
+                    {
+                      label: t('translation|Controlled Values'),
+                      getter: item => item.controlledValues,
+                    },
+                    {
+                      label: t('translation|Min Allowed'),
+                      getter: item => {
+                        let minAllowed = '';
+                        if (item.minAllowed && Object.keys(item.minAllowed).length > 0) {
+                          Object.keys(item.minAllowed).map(key => {
+                            minAllowed += `${item.minAllowed[key]},`;
+                          });
+                        }
+                        minAllowed = minAllowed.slice(0, -1);
+                        return minAllowed;
+                      },
+                    },
+                    {
+                      label: t('translation|Max Allowed'),
+                      getter: item => {
+                        let maxAllowed = '';
+                        if (item.maxAllowed && Object.keys(item.maxAllowed).length > 0) {
+                          Object.keys(item.maxAllowed).map(key => {
+                            maxAllowed += `${item.maxAllowed[key]},`;
+                          });
+                        }
+                        maxAllowed = maxAllowed.slice(0, -1);
+                        return maxAllowed;
+                      },
+                    },
+                    {
+                      label: t('translation|Mode'),
+                      getter: item => item?.mode,
+                    },
+                  ]}
+                />
+              </>
+            ),
+          },
+          {
+            id: 'headlamp.vpa-status-conditions',
+            section: (
+              <>
+                <SectionHeader title={t('translation|Conditions')} />
```

Comment:
Notice how the alignment on the sections + their titles is not correct. This can be done consistently by using the SectionBox instead of passing the SectionHeader.